build_linux.sh

```diff
@@ -17,7 +17,7 @@ for d in $PLUGINS; do
 		plugin="$(basename "$d")"
 		if [ "${plugin}" != "windows" ]; then
 			echo "  $plugin"
-			${GO:-go} build -o "${PWD}/bin/$plugin" "$@" ./"$d"
+			CGO_ENABLED=0 ${GO:-go} build -o "${PWD}/bin/$plugin" "$@" ./"$d"
```

Comment:
Can you give a rationale on why you need this?

Reply:
with that enabled, the binary would  require  glibc_2.32 to run which is not available on our machine

Reply:
I dont understand, it works on my machine :tm: 
I'm using dev-libs/glib-2.64.5

Reply:
It worked on my machine too but not the test node.

Reply:
how did you build ?

Reply:
With the flag it works both on my machine and the test node.
Without the flag it only works on my machine.
I built with both to test it out

Reply:
As seen together please remove this, and use `script/release.sh instead` :)